### DISCUSSION TITLE
Fetch API template - unable to get data on simulator

### DISCUSSION THREAD

**Discussion by juan518munoz** [zepp-health/zeppos-forum#50] [Q&A]
Greetings, I'm trying to use the fetch function, and started by downloading the sample template, I only change the json to use a Band 7 on the simulator, the button clicks, but nothing is ever fetched.

![image](https://user-images.githubusercontent.com/62400508/212552586-236d3481-48b6-4f4b-a0a9-194ee6c530db.png)

**Comment by Rafucho25** [comment]
Try this project: https://1drv.ms/u/s!AhpypwlrW5KohIB9XPwW1gJ8U2mRrQ?e=19KdkJ if this doesnt work check the log in the simulator

![image](https://user-images.githubusercontent.com/16562078/212963209-7f7e62ee-c8c0-4a00-974f-3c8329913bbe.png)

**Comment by MyoungXUE** [comment]
Hi, sorry for the long wait due to the Chinese New Year.

I ran through the experiment and the whole process was fine on my machine. I guess it could be caused by the simulator not starting correctly, I suggest restarting the simulator and running the `zeus dev` command afterwards to try again.

If there are still problems, you can provide us with some code snippets from app.json to troubleshoot the problem.

**Reply by juan518munoz** [reply]
Thank for th response, the problem had to do with how data is received, it appears to be different on the simulator than on the watch. There's a thread on the slack channel where the issue is examined further.
